images/core.proto

```diff
@@ -93,6 +95,7 @@ message thread_core_entry {
 	optional uint32			seccomp_filter	= 12;
 
 	optional string			comm		= 13;
+	optional uint64			blk_sigset_mips	= 14;
```

Comment:
Quoting from [protobuf2 specification](https://developers.google.com/protocol-buffers/docs/proto#updating):

> `optional` is compatible with `repeated`. Given serialized data of a repeated field as input, clients that expect this field to be `optional` will take the last input value if it's a primitive type field or merge all input elements if it's a message type field. 

So, I believe it's possible to change:
```
	optional uint64			blk_sigset	= 6;
```
into
```
	repeated uint64			blk_sigset	= 6;
```
and adjust the code.